workflow/infrastructure-monitoring/container-monitoring.mdx

```diff
@@ -1,73 +1,131 @@
 ---
-title: "Container Monitoring"
+title: Infrastructure Monitoring
 ---
-Overview
---------
-Container Monitoring provides real-time visibility into health and performance of containerized environments. 
 
-Container List
---------------
-The container list enables visibility into all containers across your environment excluding Kubernetes. It provides a detailed overview of your containers' health, resource consumption, and deployment. The collection of containers is container id, container name, host, status, cpu usage, ram usage, etc.
+This guide provides an overview of Middleware’s Infrastructure Monitoring page. Effortlessly collect, configure, and visualize complex telemetry data with our autodetection capabilities across your entire Infrastructure stack.
```

Comment:
Second sentence should be "and visualize complex telemetry across your Infrastructures with our autodetection capabilities". To explain - no need to use the word "data" when you use "telemetry" as telemetry is data, and pluralizing "infrastructures" is the correct grammar when talking about multiple infrastructure.